libs/shared/utils/src/index.ts

```diff
@@ -1,2 +1,5 @@
 // http
 export * from './lib/http/interceptors/auth-interceptor/auth-interceptor'
+
+// design-system
+export * from './lib/design-system/colors'
```

Comment:
I don't think this really belongs to the design system, since everything related to the design system should be in the UI library. These colors are related to charts/instances, so I think we could use something more generic

For now, you can add it to the /chart folder as chart-colors.ts and name it CHART_COLORS. We’ll see later how to manage it more cleanly

Also, the /shared library is deprecated, we’d like to remove it and replace it with more specific libraries like util-date, etc.